package.json

```diff
@@ -47,16 +48,18 @@
     "postcss-cli": "11.0.1",
     "postcss-import": "16.1.1",
     "postcss-inline-svg": "6.0.0",
+    "postcss-loader": "8.2.0",
     "postcss-preset-env": "10.5.0",
     "prettier-package-json": "2.8.0",
     "prop-types": "15.8.1",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "react-is": "18.3.1",
+    "react": "19.2.3",
+    "react-dom": "19.2.3",
+    "react-is": "19.2.3",
     "storybook": "8.6.15",
+    "style-loader": "4.0.0",
     "styled-components": "6.1.19",
     "stylelint": "16.26.1",
-    "stylelint-order": "6.0.4",
+    "stylelint-order": "7.0.0",
```

Comment:
This should be removed entirely as it is now included as a direct dependency of `@zendeskgarden/stylelint-config`